pkg/handler/http/contact.go

```diff
@@ -0,0 +1,44 @@
+package http
+
+import (
+	"context"
+	"github.com/CollActionteam/collaction_backend/internal/contact"
+	"github.com/CollActionteam/collaction_backend/internal/models"
+	"github.com/CollActionteam/collaction_backend/pkg/repository"
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+type ContactService interface {
```

Comment:
There is a ContactService and ContactHandler in the same package, what do you think about creating a layer to services and handlers?

Reply:
we can move the `ContactService` interface to the `internal/contact/email.go` file.
but by the way, I did this we defined handler needs in its territories. but both are valid.